### ISSUE TITLE
Questions about 'torch_geometric.datasets.S3DIS'

### ISSUE THREAD

**Issue by linhaojia13** [lightaime/deep_gcns_torch#19]
Hi, thank you for your good works.
I think it is convinient to use the dataloader 'torch_geometric.datasets.S3DIS' provided by torch_geometric. However, when I look into the source code for torch_geometric.datasets.s3dis
https://pytorch-geometric.readthedocs.io/en/latest/_modules/torch_geometric/datasets/s3dis.html#S3DIS , 
I foud it only download the file 'indoor3d_sem_seg_hdf5_data.zip', which is used for training but not suitable for testing. As descripted in this repo, https://github.com/charlesq34/pointnet/tree/master/sem_seg#dataset,
the file 'indoor3d_sem_seg_hdf5_data.zip' is specified for training. When in testing, one shoud download Download 3D indoor parsing dataset then transform them into blocks and execute testing by using script 'collect_indoor3d_data.py' and 'batch_inference.py'

**Comment by guochengqian** [comment]
torch_geometric.datasets.S3DIS provides the pre-processed version of S3DIS. You can set 'train=False' when you want to load the test set.
However, as you said, different methods use different ways to process the data / evaluate. PointNet evaluated them in the way you mentioned. Also, if you look at the code of PointCNN, ShellNet or KPConv, they use different evaluation methods. 
To fully reproduce our performance, we always insist on researchers using our TensorFlow version.

**Comment by linhaojia13** [comment]
Thank you for your reply. I will turn to the Tensorflow versions.
